mysql-test/main/create_trigger_after_startup.test

```diff
@@ -0,0 +1,16 @@
+--source include/not_embedded.inc
+--source include/no_view_protocol.inc
```

Comment:
I thought starting with this was standard. I will remove then.

Reply:
many test files don't have that, so, definitely not standard.

The first line makes the test to be skipped when run like `./mtr --embedded testname`.
The second line makes the test to be skipped for `./mtr --view-protocol`

Most tests run fine both in `--embedded` and in `--view`. Every case when some test doesn't needs to be analyzed, it might be a bug or might be a fundamental limitation of a feature, then you add a line like above.

I suspect that your test will work fine in `--view` (try it). But might indeed not work in `--embedded` (test it too)